iocage_lib/iocage.py

```diff
@@ -743,13 +743,24 @@ def destroy_jail(self):
         status, _ = self.list("jid", uuid=uuid)
 
         if status:
-            ioc_common.logit(
-                {
-                    "level": "INFO",
-                    "message": f"Stopping {uuid}"
-                },
-                _callback=self.callback,
-                silent=self.silent)
+            if not force:
+                ioc_common.logit(
+                    {
+                        "level": "EXCEPTION",
+                        "message": (f"Jail {uuid} is still running, "
+                                    f"please stop the jail first "
+                                    f"or destroy it with -f")
+                    }
+                )
+                return
```

Comment:
Unused return since this will trigger an exception and exit the program

Reply:
Oh, yes, will fix it.

Reply:
done